src/Activities/Internal/ActivityMetricsSender.cs

```diff
@@ -59,13 +74,21 @@ public void SendActivityMetric(Activity activity)
 
 			ReadOnlySpan<KeyValuePair<string, object?>> tagsSpan = MemoryExtensions.AsSpan(tags, 0, tagsCount);
 
+			Histogram<double> histogram = m_activityHistogram;
+			Counter<double> counter = m_activityCounter;
 			if (activity.IsHealthCheck())
 			{
-				m_healthCheckActivityCounter.Add(durationMs, tagsSpan);
+				histogram = m_healthCheckActivityHistogram;
+				counter = m_healthCheckActivityCounter;
+			}
```

Comment:
Consider using a ternary operator to assign these two variables, or an `if-else` statement, to avoid double assignment of the same variable.